integration_tests/package.json

```diff
@@ -5,8 +5,8 @@
   "repository": "https://github.com/tari-project/tari",
   "scripts": {
     "test": "cucumber-js",
-    "fmt": "prettier -w .",
-    "check-fmt": "prettier -c .",
+    "fmt": "prettier --end-of-line auto -w .",
+    "check-fmt": "prettier --end-of-line auto -c .",
```

Comment:
Was this necessary? `.prettierrc` already has it specified and I did test on windows.. 

```
$ cat .prettierrc

{
  "endOfLine": "auto"
}
```

Reply:
Yes, it was necessary, as it depends on how the source code is checked out and line endings managed with git - on my system git converts all line enfings to `CRLF` when checking out and then back to `LF` when checking in. 
`"fmt": "prettier -w ."` change all `CRLF` line endings to `LF`, thus making comparisons a nightmare.
Using `"fmt": "prettier --end-of-line auto -w ."` lets prettier conform to the majority line endings.

Reply:
Very strange, my git works the same way on windows and prettier happily picked up the setting from the `.pretterrc` config file. 🤷‍♂️